### PR TITLE
#feat - navbar and slider added.

### PR DESCRIPTION
**Pull request by surajmore98**
components added:
-navabar
-slider

### CODE REVIEW

**Comment by surajmore98** [comment]
can check preview on:
https://deploy-preview-11--charcoal-ui.netlify.app/src/documentation/component/slider.html

https://deploy-preview-11--charcoal-ui.netlify.app/src/documentation/component/navbar.html

**Review by shrey27 — APPROVED** [review]
Slider looks nice
PR is ready to merge

File: src/style/navbar.css [hunk 1]
```diff
@@ -51,9 +53,73 @@
...
+    display: block;
 }
 
 .nav-action-item:hover {
     filter: brightness(50%);
+}
+
+.nav-menu {
+    right: 0;
+    top: 3.5rem;
```
Comment (samyakshah3008) [inline src/style/navbar.css:65]: You can use shorthand property for this, can refactor later

File: src/style/slider.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  flex-direction: column;
+  gap: 2rem;
+}
+
+.slider-wrappar {
+  position: relative;
+}
+
+.slider-input {
+  width: 18rem;
```
Comment (samyakshah3008) [inline src/style/slider.css:12]: Try to give width in max or min, as for responsiveness perspective

**Review by samyakshah3008 — APPROVED** [review]
Good naming conventions 
Given some suggestions 
Clean UI 

Good for approval ✅

File: src/style/navbar.css [hunk 3]
```diff
@@ -1,16 +1,19 @@
 .nav-bar {
-    background-color: var(--charcoal-black);
-    color: var(--white);
     display: flex;
     justify-content: space-between;
     border-bottom: 1px soild var(--charcoal-white);
```
Comment (tejas-muthya37) [inline src/style/navbar.css]: looks like this typo can break your style. should be 1px solid.

**Review by tejas-muthya37 — APPROVED** [review]
Did not find any issues with the code.

3 things I liked were naming conventions, consistency in adding selectors to elements and use of CSS variables.

**Pull request merged**